apis/extension/scheduling.go

```diff
@@ -33,6 +34,31 @@ const (
 	AnnotationReservationAllocated = SchedulingDomainPrefix + "/reservation-allocated"
 )
 
+type Status string
+
+//Gang scheduling consts
+const (
+	GangAnnotationPrefix = "gang.scheduling.koordinator.sh"
+	DefaultGangWaitTime  = 2 * time.Minute
```

Comment:
the default wait time can be defined in the apis/scheduling/config. User can configure the time via the scheduler config file.